pages/api/legacy/contribute.js

```diff
@@ -0,0 +1,9 @@
+import path from 'path';
+
+export default function handler(req, res) {
+  const color = req.query.color === 'blue' ? 'blue' : 'white';
+  const verb = req.query.verb ? req.query.verb : 'contribute';
+  const size = req.query.size ? req.query.size : '';
```

Comment:
It's normally handled by the regex in rewrite, but just out of safety we could whitelist the params. That should get rid of the warning.

```suggestion
  const verb = ['donate', 'contribute'].includes(req.query.verb) ? req.query.verb : 'contribute';
  const size = req.query.size === '@2x' ? req.query.size : '';
```